code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -135,7 +135,7 @@
 
 /mob/living/silicon/robot/proc/handle_sensor_modes()
 	change_sight(removing = SEE_TURFS|SEE_MOBS|SEE_OBJS|BLIND)
-	see_invisible = SEE_INVISIBLE_LEVEL_TWO
+	see_invisible = SEE_INVISIBLE_LEVEL_ONE
```

Comment:
Have you tested this change for all potential unforeseen consequences?

Reply:
![image](https://github.com/vgstation-coders/vgstation13/assets/7573912/af70fc6e-23a1-4509-b0c7-47a205714311)

I've put myself in control of a borg and a mommi and didn't see anything wrong. We'll just have to see if players report anything unusual.